datafusion/common/src/dfschema/mod.rs

```diff
@@ -113,6 +117,10 @@ pub struct DFSchema {
     field_qualifiers: Vec<Option<TableReference>>,
     /// Stores functional dependencies in the schema.
     functional_dependencies: FunctionalDependencies,
+    /// The location in the source code where the fields are defined (e.g. in
```

Comment:
This seems somewhat strange to me as the schema may not be defined in the query

Maybe we can avoid adding this as part of V1